custom/icds_reports/static/icds_reports/js/spec/additional-filter.directive.spec.js

```diff
@@ -15,20 +15,40 @@ describe('Additional Filter Controller', function () {
     beforeEach(module('icdsApp', function ($provide) {
         $provide.constant("userLocationId", null);
         $provide.constant("isAlertActive", false);
+        $provide.constant("genders", [
```

Comment:
I think this can use `provideDefaultConstants` and pass in the appropriate options to include these